### ISSUE TITLE
how to set tab size

### ISSUE THREAD

**Issue by JOJOJOFran** [Leopotam/vscode-csharpfixformat#101]
### C# FixFormat extension version:
0.0.81

### VsCode version:
1.35.1

## Steps to reproduce
I don't khow set tab size effectively

**Comment by Leopotam** [comment]
@JOJOJOFran its part of vscode settings, not this extension: `editor.tabSize`: 2

**Comment by avin-shum** [comment]
@Leopotam It seems that the extension does not honor the vscode setting.

I have created a sample Hello World project to test: https://github.com/avin-shum/csharp-helloworld

I can change the tab size of Program.cs using C# extension but not C# FixFormat.

**Comment by Leopotam** [comment]
@avin-shum cant reproduce issue on your project - when I press "format document" - 4 spaces converted to 2 spaces, all works as must. Maybe you use another formatter? You can try to change it with "Format document with..." option from context menu and select then "C# FixFormat" option.

**Comment by Leopotam** [comment]
@JOJOJOFran is issue fixed for you?

**Comment by avin-shum** [comment]
@Leopotam I have 2 formatters installed, "C#" by Microsoft and "C# FixFormat". I reinstalled both extensions but the result is the same.

I found that when I open Program.cs, "Spaces: 4" appears at the bottom right corner. So I changed it to 2, then C# FixFormat can change the tab size.

Then I checked my Settings and found that there is a VSCode configuration "editor.detectIndentation" to detect the source indentation automatically, which is 4 for the original Program.cs. After switching this flag to false, C# FixFormat can format the document with the tab size in the Settings (2).

I cannot say that it is a bug in C# FixFormat but its behaviour is different from another formatter (i.e. "C#" by Microsoft).
